e2e/src/step_definitions/media-viewer.ts

```diff
@@ -127,6 +127,32 @@ Then('I expect to see the document should be downloaded', async function () {
   await downloadPage.waitForDownloadToComplete(file);
 });
 
```

Comment:
can we put the multimedia functions in their own file (we can separate the other functionality as we go along)
this file is getting too big, and it shouldn't have everything in it. each feature should have its own file to make it more manageable

Reply:
sure